app/controllers/katello/api/v2/content_view_versions_controller.rb

```diff
@@ -85,49 +84,6 @@ def republish_repositories
       respond_for_async :resource => task
     end
 
-    api :POST, "/content_view_versions/:id/export", N_("Export a content view version.  Relevant only for Pulp 2 repositories.")
-    param :id, :number, :desc => N_("Content view version identifier"), :required => true
-    param :export_to_iso, :bool, :desc => N_("Export to ISO format."), :required => false
-    param :iso_mb_size, :number, :desc => N_("maximum size of each ISO in MB. Relevant only for Pulp 2 repositories"), :required => false
-    param :since, Date, :desc => N_("Optional date of last export (ex: 2010-01-01T12:00:00Z). Relevant only for Pulp 2 repositories"), :required => false
-    def export
-      if SmartProxy.pulp_primary.pulp3_repository_type_support?(Katello::Repository::YUM_TYPE)
-        fail HttpErrors::BadRequest, _("Invalid usage for Pulp 3 repositories. "\
-                                       "Use hammer content-export for Yum repositories")
-      end
-      ::Foreman::Deprecation.api_deprecation_warning("Export is being deprecated and will be removed in a future version of Katello. Use hammer content-view version export instead.")
-      if params[:export_to_iso].blank? && params[:iso_mb_size].present?
-        fail HttpErrors::BadRequest, _("ISO export must be enabled when specifying ISO size")
-      end
-
-      if (repos = @content_view_version.content_view.on_demand_repositories).any?
-        fail HttpErrors::BadRequest, _("This content view has on demand repositories that cannot be exported: %{repos}" % {repos: repos.pluck(:label).join(", ")})
-      end
-
-      if params[:since].present?
-        begin
-          params[:since].to_datetime
-        rescue
-          raise HttpErrors::BadRequest, _("Invalid date provided.")
-        end
-      end
-      task = async_task(::Actions::Katello::ContentViewVersion::Export, @content_view_version,
```

Comment:
Drive-by review! Are this action and the below ::Import still used?

Reply:
Good catch, @parthaa do you know if we still use this?

Reply:
`git grep ContentViewVersion::Export` should give a pretty clear answer :)

Reply:
Haha fair, ok ill unping Partha

Reply:
Looks like all this stuff is still being used in places, for now will just close this and go with the hammer pr